fe/fe-core/src/main/java/com/starrocks/http/BaseAction.java

```diff
@@ -306,9 +342,28 @@ protected void checkTblAuth(UserIdentity currentUser, String db, String tbl, Pri
         }
     }
 
+    protected void checkTableAction(ConnectContext context, String db, String tbl,
+                                    PrivilegeType.TableAction action) throws UnauthorizedException {
+        if (!PrivilegeManager.checkTableAction(context, db, tbl, PrivilegeType.TableAction.INSERT)) {
```

Comment:
Forgot to use the action parameter? Wrong use of PrivilegeType.TableAction.INSERT?

Reply:
yes, my mistake, fixed